bootstrap/cmd/bootstrap/app/kfctlServer.go

```diff
@@ -246,6 +245,15 @@ func (s *kfctlServer) handleDeployment(r kfdefsv3.KfDef) (*kfdefsv3.KfDef, error
 		}
 	}
 
+	log.Infof("Calling apply Patch")
+	if err := s.kfApp.Apply(kftypes.PATCH); err != nil {
```

Comment:
is this analog to `kubectl apply`?  could you point me to the code implementing this patch?

Reply:
This point to the new piece in `coordinator.go` in this PR.